### DISCUSSION TITLE
Sim Slot Bug

### DISCUSSION THREAD

**Discussion by Oredola** [romellfudi/VoIpUSSD#72] [Announcements]
Hello,
Thank you for your great work. I have a problem with the library. Whatever the position of the sim we are, the ussd code is always executed on the first SIM. Please help me solve this problem. Thank you

**Comment by romellfudi** [comment]
Howdy @Oredola 
Could you please update your logcat?
use the `USSDService` tag.

**Comment by romellfudi** [comment]
This issue was solved in #86.
Please download last [version](  https://github.com/romellfudi/VoIpUSSD/releases/tag/Rev28)

**Comment by romellfudi** [comment]
Enjoy it updating your gradle files by using [jitpack release]( https://jitpack.io/#romellfudi/VoIpUSSD)
